tests/ui/specs/Popup/Popup-specs.js

```diff
@@ -193,1266 +192,1248 @@ describe('Popup', function () {
 
 		const popup = Page.components.popup2;
 
-		it('should have correct title', function () {
-			Page.waitTransitionEnd(3000, undefined, () => {
+		it('should have correct title', async function () {
+			await Page.waitTransitionEnd(3000, undefined, async () => {
 				popupCommon.buttonPopup2.click();
 			});
-			expectOpen(popupCommon);
-			validateTitle(popup, 'Popup without AutoDismiss');
+			await expectOpen(popupCommon);
+			await validateTitle(popup, 'Popup without AutoDismiss');
 		});
 
 		describe('5-way', function () {
-
-			it('should spot default button in popup container', function () {
-				Page.spotlightRight();
-				Page.waitTransitionEnd(3000, 'popup open', () => {
-					Page.spotlightSelect();
-				});
-				expectOpen(popupCommon);
-				expect(popup.buttonOK.isFocused()).to.be.true();
-			});
-
-			it('should not dismiss the popup and should not move spotlight from the popup container', function () {
-				Page.spotlightRight();
-				Page.waitTransitionEnd(3000, 'popup open', () => {
+			it('should spot default button in popup container', async function () {
+				await Page.spotlightRight();
+				// await Page.waitTransitionEnd(3000, 'popup open', async () => {
+				// 	await Page.spotlightSelect();
+				// });
+				await Page.spotlightSelect();
+				await browser.pause(3000);
```

Comment:
Can't we use waitTransitionEnd?  Can't it be like line 217?
Because it aways wait 3sec

Reply:
sorry, i missed this. i pushed a fix